_opam-files/lwt-core.opam

```diff
@@ -1,30 +1,24 @@
 opam-version: "1.2"
-name: "lwt_ssl"
-version: "1.0.1"
+name: "lwt"
```

Comment:
`lwt-core` might be too confusing with `core-lwt` already a package in OPAM, but we can think about that later.